src/Pages/Music/Videos/Videoid_Int.elm

```diff
@@ -0,0 +1,111 @@
+module Pages.Music.Videos.Videoid_Int exposing (Model, Msg, Params, page)
+
+import Element exposing (..)
+import Shared exposing (sendActions)
+import Spa.Document exposing (Document)
+import Spa.Page as Page exposing (Page)
+import Spa.Url exposing (Url)
+import WSDecoder exposing (AlbumObj, ArtistObj, SongObj, VideosObj)
+
+
+page : Page Params Model Msg
+page =
+    Page.application
+        { init = init
+        , update = update
+        , subscriptions = subscriptions
+        , view = view
+        , save = save
+        , load = load
+        }
+
+
+
+-- INIT
+
+
+type alias Params =
+    { videoid : Int }
+
+
+type alias Model =
+    { artist_list : List ArtistObj
+    , videoid : Int
+    , videos : Maybe VideosObj
+    , album_list : List AlbumObj
+    , videos_list : List VideosObj
+    }
+
+
+init : Shared.Model -> Url Params -> ( Model, Cmd Msg )
+init shared { params } =
+    ( { videoid = params.videoid
+      , videos = getVideos params.videoid shared.videos_list
+      , album_list = shared.album_list
+      , artist_list = shared.artist_list
+      , videos_list = shared.videos_list
+      }
+    , Cmd.none
+    )
+
+
+checkVideoId : Int -> VideosObj -> Bool
+checkVideoId firstId video =
```

Comment:
I think this can be safely inlined and probably be even nicer that way

Reply:
I was not able to do this as elm-format discourages one-liner if statements. After running elm format it changes back to the previous.

Reply:
I don't think you got what  I wanted you to do :)

I would move this to an anonymous inlined function

Reply:
Basically remove this function and use this instead in line 63 
`List.head (List.filter (\video -> id == video.videoid) videolist)`